scripts/vagrant_setup.sh

```diff
@@ -10,12 +10,13 @@ sudo mkdir -p /srv/salt
 sudo mkdir -p /srv/pillar
 sudo mkdir -p /srv/formulas
 sudo cp $SRCDIR/pillar.example /srv/pillar/pillar.sls
-sudo cp -r $SRCDIR/rabbitmq /srv/salt
+sudo ln -s $SRCDIR/rabbitmq /srv/salt/rabbitmq
```

Comment:
Hah, maybe we should be doing this for our other formula dev environments. I saw that you also turned on rsync.